source/event_stream_rpc_client_continuation.c

```diff
@@ -175,25 +175,27 @@ PyObject *aws_py_event_stream_rpc_client_continuation_activate(PyObject *self, P
         return NULL;
     }
 
+    /* From hereon, we need to clean up if errors occur */
+
+    bool success = false;
```

Comment:
This is never used

Reply:
Unless this is an artistic commentary on software. In which case, leave it.

Reply:
It's used:
```
if (success) {
        Py_RETURN_NONE;
    }

    /* failed */
    Py_DECREF(on_flush_py);
    if (self_py_set) {
        Py_CLEAR(continuation->self_py);
    }

    return NULL;
    ```